### PR TITLE
Added response

### PR DESCRIPTION
**Pull request by sfaaland**
Better communication

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1 +1,2 @@
 print("Hello, world!\n")
+print("Goodby, nerd!\n")
```
Comment (ofaaland) [inline hello.py:2]: What do you mean, "nerd"?!

**Review by ofaaland — CHANGES_REQUESTED** [review]
Come up with a better term

File: hello.py (hunk 1 above)
Comment (sfaaland) [reply hello.py:2]: You know it's true.

Comment (ofaaland) [reply hello.py:2]: Yeah, you're right

**Review by ofaaland — APPROVED** [review]
LGTM

**Pull request merged**